docs/references/api/deviceshifu-configmap.md

```diff
@@ -79,6 +80,19 @@ instructions: |
       ... # You can continue to configure commands and corresponding NodeID according to your own needs, just continue to add according to this format
 ```
 
+- **Gateway**
+
+```yml
+instructions: |
+    instructionSettings:
+      defaultTimeoutSeconds: 8
+    instructions:
+      read_value:
+        gatewayPropertyList:
```

Comment:
this should be a separate section since it is gateway property not protocol property